docs/firmware/bom-stm32node/build.md

```
@@ -5,7 +5,34 @@
+    ```bash
+    cd fw/common/boomlink && task setup
+    ```
+
+    `task build` in `fw/bom-stm32node` puts that venv on `PATH` automatically. If you
+    prefer the raw `cmake` invocation below, activate it yourself first
+    (`source ../common/boomlink/.venv/bin/activate`) or make sure your `python3` has
+    both packages.
+
+Then, from `fw/bom-stm32node`:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)